graphql/resolvers/exerciseCommentCrud.ts

```diff
@@ -44,3 +45,28 @@ export const addExerciseComment = async (
     data: { authorId, content, exerciseId, parentId, userPic }
   })
 }
+
+export const editExerciseComment = async (
```

Comment:
Can't we wrap this with [WithUserContainer](https://github.com/garageScript/c0d3-app/blob/7ab5efcd9c2b726846cc747ca83e846712ab27cf/containers/withUserContainer.ts) higher order function?